src/main/java/uk/gov/hmcts/dts/fact/controllers/admin/list/AdminOpeningTypeController.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.dts.fact.controllers.admin.list;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.apache.commons.lang.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.hmcts.dts.fact.config.security.Role;
+import uk.gov.hmcts.dts.fact.model.admin.OpeningType;
+import uk.gov.hmcts.dts.fact.services.admin.list.AdminOpeningTypeService;
+
+import java.net.URI;
+import java.util.List;
+
+import static org.springframework.http.ResponseEntity.created;
+import static org.springframework.http.ResponseEntity.ok;
+import static uk.gov.hmcts.dts.fact.services.admin.AdminRole.FACT_ADMIN;
+import static uk.gov.hmcts.dts.fact.services.admin.AdminRole.FACT_SUPER_ADMIN;
+
+@RestController
+@RequestMapping(
+    path = "/admin/openingTypes",
+    produces = {MediaType.APPLICATION_JSON_VALUE}
+)
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+public class AdminOpeningTypeController {
+
+    private final AdminOpeningTypeService adminService;
+
+    @Autowired
+    public AdminOpeningTypeController(AdminOpeningTypeService adminService) {
+        this.adminService = adminService;
+    }
+
+
+    @GetMapping()
+    @ApiOperation("Retrieve all opening types")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Successful", response = OpeningType.class, responseContainer = "List"),
+        @ApiResponse(code = 401, message = "Unauthorized"),
+        @ApiResponse(code = 403, message = "Forbidden")
+    })
+    @Role({FACT_ADMIN, FACT_SUPER_ADMIN})
```

Comment:
On the card it mentions an A/C that for getting all the opening types, that only a super user can do this? 

Scenario 1 - view list of opening types

Given I am a super admin user

When I navigated to the opening types tab of the Lists page

Then I am presented with a list of the opening types

Reply:
fact-admin will be accessing all opening types from edit court Opening Hours tab so both roles are required